copier.yml

```diff
@@ -111,6 +111,5 @@ _skip_if_exists:
     - README.md
     - .gitattributes
 
-# Tasks to execute after generating or updating a project.
-_tasks:
-    - "git init --initial-branch=main"
+# Require that the user has at least Copier v8.0.0 installed
+_min_copier_version: "8.0.0"
```

Comment:
I trust copier issues a sensible error message if the version is old?

Reply:
Yeah, it's pretty clean: 

```
(scratch) drew@allosaurus:~/code$ copier --vcs-ref=HEAD ./python-project-template/ science
This template requires Copier version >= 8.0.0, while your version of Copier is 7.2.0.
```